### PR TITLE
docs+examples: Add Signoz

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR adds SigNoz:

- Example app with Pokeshop
- Recipe

This PR updates:

- Docs config

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: examples/tracetest-signoz-pokeshop/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Tracetest + SigNoz + Pokeshop
+
+<!-- > [Read the detailed recipe for setting up Tracetest + SigNoz + Pokeshop in our documentation.]() -->
+
+This examples' objective is to show how you can:
+
+1. Configure Tracetest to ingest traces.
+2. Configure SigNoz and use it as a trace data store.
+3. Configure SigNoz to query traces
```
Comment (jfermi) [inline examples/tracetest-signoz-pokeshop/README.md:9]:
```suggestion
3. Configure SigNoz to query traces.
```

File: examples/tracetest-signoz-pokeshop/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Steps
+
+1. [Install the tracetest CLI](https://docs.tracetest.io/installing/)
```
Comment (jfermi) [inline examples/tracetest-signoz-pokeshop/README.md:13]:
```suggestion
1. [Install the tracetest CLI.](https://docs.tracetest.io/installing/)
```

File: examples/tracetest-signoz-pokeshop/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+2. Run `tracetest configure --endpoint http://localhost:11633` on a terminal
```
Comment (jfermi) [inline examples/tracetest-signoz-pokeshop/README.md:14]:
```suggestion
2. Run `tracetest configure --endpoint http://localhost:11633` on a terminal.
```

File: examples/tracetest-signoz-pokeshop/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+3. Run the project by using docker-compose: `docker-compose up -d` (Linux) or `docker compose up -d` (Mac)
```
Comment (jfermi) [inline examples/tracetest-signoz-pokeshop/README.md:15]:
```suggestion
3. Run the project by using docker-compose: `docker-compose up -d` (Linux) or `docker compose up -d` (Mac).
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by danielbdias — APPROVED** [review]

**Review by xoscar — APPROVED** [review]

**Pull request merged**